tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py

```diff
@@ -43,6 +43,8 @@
 
 
 class Cert_5_3_10_AddressQuery(thread_cert.TestCase):
+    support_ncp = False
```

Comment:
```suggestion
    SUPPORT_NCP = False
```

Reply:
this will be done in a separate PR together with the topology attribute.

Reply:
fine to apply style changes in a separate PR